app/jobs/resolution_proofing_job.rb

```diff
@@ -48,8 +45,8 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
                           )
                         end
 
-    if use_lexisnexis_ddp_threatmetrix_before_rdp_instant_verify?
-      add_threatmetrix_result_to_callback_result(callback_log_data.result, threatmetrix_result)
+    unless optional_threatmetrix_result.nil?
```

Comment:
another check I think would read better as a positive check
```suggestion
    if optional_threatmetrix_result.present?
```